README.md

```diff
@@ -58,3 +58,15 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
```

Comment:
Your README could also mention Vagrant - something along the lines of "Alternatively, you can run the project inside a VM if you ..."